### PR TITLE
get_machine_metrics_cAdvisor + report_machine_metrics_cAdvisor Test

### PR DESCRIPTION
**Pull request by Bonczidai**
Signed-off-by: Bonczidai Levente <levente.bonczidai@hp.com>

### CODE REVIEW

**Comment by Bonczidai** [comment]
what I did here:

- cadvisor flows require a running cadvisor container - I create it in the circleci config because:
  - circleci does not currently enable container removal, so we cannot start up and remove the container in every flow
  - all flows can use the same container
- cadvisor flows should be run only on circleci: added `cadvisor` test suite

File: circle.yml [hunk 1]
```diff
@@ -22,14 +22,15 @@ dependencies:
...
+  pre:
+    - docker run --privileged -d --name cadvisor --volume=/:/rootfs:ro --volume=/var/run:/var/run:rw --volume=/sys:/sys:ro --volume=/var/lib/docker/:/var/lib/docker:ro --volume=/cgroup:/cgroup --publish=32951:8080 google/cadvisor:latest
 #    - docker run --privileged --lxc-conf="lxc.aa_profile=unconfined" -d -p 4444 -p 49153:22 -e PORT=4444 --name docker_host_ssh orius123/dind-ssh
 #    - docker port docker_host_ssh 22
   override:
     - wget https://github.com/CloudSlang/cloud-slang/releases/download/cloudslang-0.7.28/cslang-builder.zip
     - unzip cslang-builder.zip -d cslang-builder
     - chmod +x cslang-builder/bin/cslang-builder
-    - ./cslang-builder/bin/cslang-builder
+    - ./cslang-builder/bin/cslang-builder -ts cadvisor
```
Comment (orius123) [inline circle.yml]: we can't merge it before we make builder run also tests with no TS when the user define to run with cadvisor for instance. or that only our cadvisor and docker tests will run

**Comment by orius123** [comment]
pending CloudSlang/cloud-slang#296

**Comment by orius123** [comment]
@Bonczidai is this dependedant on #154 ?

**Comment by Bonczidai** [comment]
no, this is not dependent on #154

**Comment by Bonczidai** [comment]
- report machine metrics test - also covers: get_machine_metrics_cAdvisor and parse_cadvisor_machine

- [cadvisor] test suite is specified so it will not be started on travis, only on circle ci

- now the cadvisor container is started in the flow (need to see future implications regarding this)

**Comment by orius123** [comment]
how does that pass and when we try locally we can't get any details?

**Comment by Bonczidai** [comment]
well, this operation uses `/api/v1.2/machine` (machine specific info) url which works, the other PR uses `/api/v1.2/docker/container_name` (docker specific info)

**Pull request merged**